docs/source/user_guide/getting_started_with_dpf_server.rst

```diff
@@ -7,46 +7,52 @@ Getting started with DPF Server
 What is DPF Server
 ------------------
 
-The Data Processing Framework (DPF) provides numerical simulation users and engineers with a toolbox for accessing and transforming 
```

Comment:
@PipKat I do not think we give the meaning behind the acronym anywhere else. Maybe we could keep it?